.github/workflows/ci.yml

```diff
@@ -104,7 +104,9 @@ jobs:
         uses: actions/upload-artifact@v2
         with:
           name: manylinux-artifacts
-          path: target/wheels/*manylinux*.whl
+          path: |
+            target/wheels/*manylinux_2_17*.whl
```

Comment:
Just to double check, is this version number informed by a specific version in another part of the repo, like a script or Dockerfile? I didn't see it anywhere else on quick perusal, so wanted to make sure it wasn't labeling something as a specific version when that version might actually change.

Reply:
@swernli This version is held by glibc inside of the manylinux container. As we also build on Ubuntu 20.04, it will generate wheels matching `*manylinux_2_31*.whl` which we don't want to distribute. Easier to allowlist than blocklist as versions update.